Index: jessie-stage3/rootfs/lib/systemd/system/nginx.service
===================================================================
--- jessie-stage3.orig/rootfs/lib/systemd/system/nginx.service
+++ jessie-stage3/rootfs/lib/systemd/system/nginx.service2
@@ -17,12 +17,13 @@
 [Service]
 Type=forking
 PIDFile=/run/nginx.pid
-ExecStartPre=/usr/sbin/nginx -t -q -g 'daemon on; master_process on;'
-ExecStart=/usr/sbin/nginx -g 'daemon on; master_process on;'
-ExecReload=/usr/sbin/nginx -g 'daemon on; master_process on;' -s reload
+ExecStartPre=/usr/sbin/nginx -t -q -g 'daemon on; master_process on;' -c /data/conf/nginx/nginx.conf
+ExecStart=/usr/sbin/nginx -g 'daemon on; master_process on;' -c /data/conf/nginx/nginx.conf
+ExecReload=/usr/sbin/nginx -g 'daemon on; master_process on;' -c /data/conf/nginx/nginx.conf -s reload
 ExecStop=-/sbin/start-stop-daemon --quiet --stop --retry QUIT/5 --pidfile /run/nginx.pid
 TimeoutStopSec=5
 KillMode=mixed
 
 [Install]
 WantedBy=multi-user.target
